oidc-ui/src/components/Consent.js

```diff
@@ -244,11 +245,19 @@ export default function Consent({
   }, []);
 
   function formatTime(time) {
-    const minutes = Math.floor(time / 60).toString().padStart(2, "0");
+    const minutes = Math.floor(time / 60)
+      .toString()
+      .padStart(2, "0");
     const seconds = (time % 60).toString().padStart(2, "0");
     return `${minutes}:${seconds}`;
   }
 
+  useEffect(() => {
+    if (authTime === null) {
+      onError("invalid_transaction", t("invalid_transaction"));
+    }
+  }, []);
```

Comment:
we already have a useEffect with Empty array. Why are we required to add one more. can this logic be moved to line 238?

Reply:
The logic on the line 238 ie., isNaN(authTime) will return false (on console.log()) all the time irrespective of the authTime